raiden/raiden_service.py

```diff
@@ -255,6 +251,12 @@ def start(self):
                 block_number,
             )
             self.wal.log_and_dispatch(state_change, block_number)
+            payment_network = PaymentNetworkState(
+                self.default_registry.address,
+                [],  # empty list of token network states as it's the node's startup
```

Comment:
here is the same thing as the token network, we can just set it to the empty list in the constructor

Reply:
This argument is still used in the `maybe_add_token_network()` function [here](https://github.com/raiden-network/raiden/blob/da745f5eff87f4e81ed7b2be5a2ff5cc1d76c359/raiden/transfer/node.py#L349)

Reply:
Ahh, okay